cardano-cli/src/Cardano/CLI/Pretty.hs

```diff
@@ -0,0 +1,72 @@
+module Cardano.CLI.Pretty
+  ( Ann,
```

Comment:
Can you fix the export style to follow the usual style in `cardano-cli`?
```
( Ann
, ...
```

Reply:
Done!